lib/modules/platform/types.ts

```diff
@@ -163,6 +163,10 @@ export type EnsureCommentRemovalConfig =
 
 export type EnsureIssueResult = 'updated' | 'created';
 
+export interface AutodiscoverParams {
```

Comment:
```suggestion
export interface AutodiscoverConfig {
```
we mostly use config as name